examples/next-app/src/app/page.tsx

```diff
@@ -0,0 +1,28 @@
+'use client'
+import { Database, fireproof } from "@fireproof/core";
+import { connect } from "@fireproof/ipfs"
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+
+type AuthFormValues = {
+  email: string
+}
+
+export default function Home () {
+  const [db] = useState<Database>(fireproof('example'))
```

Comment:
I tried to make it unnecessary to wrangle details like this for users of the React hooks. See eg this where the `id` is the database name, taken from the url path slug. I'm not sure it's right, but wanna make sure you grok that approach early on and give me feedback: https://github.com/fireproof-storage/firehouse-chat/blob/bc8cf00df8cb242456ddef3ffd84bea24faf7fe3/src/components/Channel.tsx#L44